--- a/../python-cmake-buildsystem-master/CMakeLists.txt	2020-05-20 16:55:32.388103600 -0400
+++ b/../python-cmake-buildsystem-master/CMakeLists.txt	2020-05-20 16:56:13.286449300 -0400
@@ -356,9 +356,6 @@
     set(EXTENSION_INSTALL_DIR DLLs)
 endif()
 set(INCLUDE_INSTALL_DIR include/python${LD_VERSION})
-if(MSVC)
-    set(INCLUDE_INSTALL_DIR include)
-endif()
 # Build tree directory
 set(BIN_BUILD_DIR ${PROJECT_BINARY_DIR}/bin)
 set(CONFIG_BUILD_DIR ${PROJECT_BINARY_DIR}/${CONFIG_INSTALL_DIR})
