--- hdf5-1.8.20/CMakeLists.txt	2018-05-26 18:01:08.243573951 -0700
+++ hdf5-1.8.20/CMakeLists.txt	2018-05-26 18:00:02.000000000 -0700
@@ -631,7 +631,7 @@
       message (STATUS " **** Allowing unsupported HL and thread-safety options **** ")
     endif ()
   endif ()
-  if (H5_HAVE_IOEO)
+  if (WIN32) #if (H5_HAVE_IOEO)
     message (STATUS " **** Win32 threads requires WINVER>=0x600 (Windows Vista/7/8) **** ")
     set (H5_HAVE_WIN_THREADS 1)
   else ()
